### PR TITLE
EVG-8041 host/volume expiration reminder improvements

### PR DESCRIPTION
**Pull request by ybrill**
- format the expiration time and localize the time zone
- link to the host/volume page with the host/volume selected

### CODE REVIEW

File: trigger/host.go [hunk 1]
```diff
@@ -181,13 +183,22 @@ func hostExpirationSlackPayload(t hostTemplateData, messageString string, select
...
 		}},
 	}, nil
 }
 
 func (t *hostTriggers) hostExpiration(sub *event.Subscription) (*notification.Notification, error) {
-	return t.generate(sub, expiringHostTitle, expiringHostBody)
+	timeZone := time.Local
+	if sub.OwnerType == event.OwnerTypePerson {
+		userTimeZone, err := getUserTimeZone(sub.Owner)
+		grip.Error(errors.Wrap(err, "problem getting time zone"))
```
Comment (john-m-liu) [inline trigger/host.go:197]: I would use message.Fields to make this more easily searchable

File: trigger/volume.go [hunk 2]
```diff
@@ -104,5 +107,31 @@ func (t *volumeTriggers) generate(sub *event.Subscription, subjectTempl, bodyTem
 }
 
 func (t *volumeTriggers) volumeExpiration(sub *event.Subscription) (*notification.Notification, error) {
-	return t.generate(sub, expiringVolumeTitle, expiringVolumeBody)
+	timeZone := time.Local
+	if sub.OwnerType == event.OwnerTypePerson {
+		userTimeZone, err := getUserTimeZone(sub.Owner)
+		grip.Error(errors.Wrap(err, "problem getting time zone"))
```
Comment (john-m-liu) [inline trigger/volume.go:113]: message.Fields here as well

File: trigger/host.go (hunk 1 above)
Comment (ybrill) [reply trigger/host.go]: done

File: trigger/volume.go (hunk 2 above)
Comment (ybrill) [reply trigger/volume.go]: done

**Pull request merged**